docs/node/migrations/zero-downtime-migrations/state-replication.mdx

```diff
@@ -0,0 +1,68 @@
+---
+sidebar_position: 2
+sidebar_label: "State Replication"
+id: state-replication
+title: State Replication
+description: State Replication
+slug: /node/migrations/zero-downtime-migrations/state-replication
+---
+
+Zero-Downtime Migrations require the new network to start with the state of the existing network at the start of the migration and then replicate the state changes of the existing network to the new network during the migration period to stay in sync with the existing network.
+
+## State Replication
+
+State replication is handled by an inbuilt `migrations` listener extension in the Kwil node. The `migrations` listener extension polls for the state changes on the existing network and replicates the changes onto the new network. The `migrations` listener extension can be enabled by setting the below configuration in the `config.toml` and `genesis.json` files.
+
+
+`migrate_from` config in the `config.toml` file specifies the JSON-RPC address of the node from the existing network that the new network should replicate the state changes from. 
+```toml
+[app]
+"migrate_from" = "http://kwild-jsonrpc-server-address:8484"
```

Comment:
No quotes around `migrate_from` I believe.

Should also mention that the connection should be secure to ensure the received data is not tampered with.